src/main/java/mate/academy/hibernate/relations/dao/impl/ActorDaoImpl.java

```diff
@@ -2,8 +2,11 @@
 
 import java.util.Optional;
 import mate.academy.hibernate.relations.dao.ActorDao;
+import mate.academy.hibernate.relations.exception.DataProcessingException;
```

Comment:
Ensure that the `DataProcessingException` class is correctly defined in the `exception` package. This custom exception should extend `RuntimeException` or another appropriate exception class.